tests/portfolio/test-portfolio-solver.cpp

```diff
@@ -0,0 +1,117 @@
+/*********************                                                        */
+/*! \file test-generic-solver.cpp
+** \verbatim
+** Top contributors (to current version):
+**   Amalee Wilson
+** This file is part of the smt-switch project.
+** Copyright (c) 2020 by the authors listed in the file AUTHORS
+** in the top-level source directory) and their institutional affiliations.
+** All rights reserved.  See the file LICENSE in the top-level source
+** directory for licensing information.\endverbatim
+**
+** \brief
+**
+**
+**/
+
+#include "assert.h"
+#include "boolector_factory.h"
+#include "cvc4_factory.h"
+#include "msat_factory.h"
+#include "portfolio_solver.h"
+#include "yices2_factory.h"
+
+using namespace smt;
+using namespace std;
+
+int main()
+{
+  cout << "testing portfolio solver" << endl;
+  SmtSolver s = MsatSolverFactory::create(false);
+  s->set_opt("produce-models", "true");
+  Sort intsort = s->make_sort(BV, 10);
```

Comment:
minor: but I usually follow a naming convention where `intsort` would refer to an integer sort, and for bit vectors I typically use `bvsort` or `bvsort<size>`.

Reply:
Oops, forgot to fix that when I changed it.